### PR TITLE
Add documentation for quantize feature

### PR DESCRIPTION
**Pull request by trappar**
I had trouble finding the documentation on this feature so I figured I'd add something to the community features so we don't lose track of it!

### CODE REVIEW

File: Documentation/community_features.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ Synchronization modes accessible through the "LFO SYNC" shortcut.
 ### Kit Keyboard View
  - ([#112]) All-new use for the "keyboard" button in kit clips, uses the main pad grid for MPC-style 16 level playing. Horizonatal encoder scrolls by one pad at a time, allowing positioning drums left to right, and vertical encoder jumps vertically by rows.
 
+### Quantize and Humanize Notes
```
Comment (bobtwinkles) [inline Documentation/community_features.md:60]: The headers are intended to be the view where this is possible, so maybe "All Clip Views"?

File: Documentation/community_features.md [hunk 2]
```diff
@@ -57,6 +57,9 @@ Synchronization modes accessible through the "LFO SYNC" shortcut.
...
+  - ([#129]) Press and hold a note in clip view and turn the tempo knob right or left to apply quantize or humanize respectively to that row. Pressing and turning the tempo knob will apply quantize or humanize to all rows. The amount of quantization/humanization is shown in the display. This feature can be toggled in the [runtime features menu](#runtime-features).
```
Comment (bobtwinkles) [inline Documentation/community_features.md:61]: You'll need to add this to the list of PRs at the end of the file for github to render the link correctly.

**Review by bobtwinkles — COMMENTED** [review]
Thanks! 2 minor comments but otherwise LGTM

File: Documentation/community_features.md (hunk 1 above)
Comment (trappar) [reply Documentation/community_features.md]: Updated with that in mind.

**Review by bobtwinkles — APPROVED** [review]

**Pull request merged**